cmd/buildtools/k0s.go

```diff
@@ -92,58 +135,35 @@ var updateK0sImagesCommand = &cli.Command{
 			}
 
 			if err := SetMakefileVariable(component.makefileVar, fmt.Sprintf("%s@%s", packageVersion, digest)); err != nil {
-				return fmt.Errorf("failed to set %s version: %w", component.name, err)
+				return fmt.Errorf("failed to set %s version: %w", componentName, err)
 			}
 		}
 
 		return nil
 	},
 }
 
-func makeK0s(k0sVersion string) error {
-	if k0sVersion != "" {
-		cmd := exec.Command("make", "pkg/goods/bins/k0s", fmt.Sprintf("K0S_VERSION=%s", k0sVersion), "K0S_BINARY_SOURCE_OVERRIDE=")
```

Comment:
why lose this?

Reply:
you mean `K0S_BINARY_SOURCE_OVERRIDE=`?

Reply:
yes

Reply:
if we're overriding the k0s binary, why wouldn't we want to get the images from the custom binary?

Reply:
but it makes no sense when passing in k0s_verison as input

Reply:
how so? custom k0s binaries aren't downloadable from the k0s github project, only via the `K0S_BINARY_SOURCE_OVERRIDE` url.

so this workflow would break if we ever set `K0S_VERSION` to a custom build version.

Reply:
i guess the conflict is that i'm reading the `K0S_VERSION` from the Makefile if it's not passed as an input: https://github.com/replicatedhq/embedded-cluster/pull/817/files#diff-b7c04cb7731e67d487af4a6bf8367637e1da8e995aaf361cae5c41dc110eae90R79

maybe i should change that

Reply:
but i use that version is used later on. hmm. i'll think about it

Reply:
will fix on the following pr